catalog/security/catalog-security-pointofcontact-policyplugin/src/main/java/org/codice/ddf/catalog/security/policy/metacard/PointOfContactPolicyPlugin.java

```diff
@@ -0,0 +1,126 @@
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.operation.OperationTransaction;
+import ddf.catalog.operation.Query;
+import ddf.catalog.operation.ResourceRequest;
+import ddf.catalog.operation.ResourceResponse;
+import ddf.catalog.plugin.PolicyPlugin;
+import ddf.catalog.plugin.PolicyResponse;
+import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.plugin.impl.PolicyResponseImpl;
+import ddf.security.permission.Permissions;
+
+public class PointOfContactPolicyPlugin implements PolicyPlugin {
+    private static final String[] PERMISSION_STRING = {"read-only=Cannot update the point-of-contact field"};
+
+    private static final Map<String, Set<String>> PERMISSION_MAP =
+            Permissions.parsePermissionsFromString(PERMISSION_STRING);
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PointOfContactPolicyPlugin.class);
+
+    @Override
+    public PolicyResponse processPreCreate(Metacard input, Map<String, Serializable> properties)
+            throws StopProcessingException {
+        return new PolicyResponseImpl();
+    }
+
+    @Override
+    public PolicyResponse processPreUpdate(Metacard newMetacard,
+            Map<String, Serializable> properties) throws StopProcessingException {
+        List<Metacard> previousStateMetacards = ((OperationTransaction) properties.get(
+                OPERATION_TRANSACTION_KEY)).getPreviousStateMetacards();
+
+        Metacard previous;
+        try {
+            previous = previousStateMetacards.stream()
+                    .filter((x) -> x.getId()
+                            .equals(newMetacard.getId()))
+                    .findFirst()
+                    .get();
+        } catch (NoSuchElementException e) {
+            LOGGER.debug("Cannot locate metacard {} for update.",
+                    newMetacard.getId());
+            return new PolicyResponseImpl();
+        }
+
+        return pointOfContactChanged(newMetacard, previous) ? new PolicyResponseImpl(null,
+                PERMISSION_MAP) : new PolicyResponseImpl();
+    }
+
+    private boolean pointOfContactChanged(Metacard newMetacard, Metacard previousMetacard) {
+        Attribute newPointOfContact = newMetacard.getAttribute(Metacard.POINT_OF_CONTACT);
+        Attribute oldPointOfContact = previousMetacard.getAttribute(Metacard.POINT_OF_CONTACT);
+
+        if (newPointOfContact != null && oldPointOfContact != null) {
+            return !newPointOfContact.getValue()
+                    .equals(oldPointOfContact.getValue());
+        } else {
+            //Return true if only one of them is null
+            return newPointOfContact != null ^ oldPointOfContact != null;
```

Comment:
❓ Nice use of the xor operator here! So rarely used that I had to look it up :-)
Another (simpler?) option would be to do `return newPointOfContact != oldPointOfContact`.
We could remove the `else` block too.